### ISSUE TITLE
Not building on latest nightly

### ISSUE THREAD

**Issue by theduke** [valeriansaliou/rs-card-validate#1]
```
/home/theduke/.cargo/registry/src/github.com-1ecc6299db9ec823/card-validate-2.1.1/src/lib.rs:104:38
    |
104 |             Type::VisaElectron => (16...16),
    |                                      ^^^
    |
    = help: Use `..` if you need an exclusive range (a < b)
    = help: or `..=` if you need an inclusive range (a <= b)
```

**Comment by valeriansaliou** [comment]
Which version of the Rust compiler are you using, and which channel (stable / beta / nightly)?

**Comment by valeriansaliou** [comment]
Woops forget about that, didn't see the title (latest nighly). I'll look into it.

**Comment by valeriansaliou** [comment]
Just released 0505019870d4eb789cecc35dbb27c495bcfefa4c (v2.1.2) — Can you update your dep. and try again? (I removed the `inclusive_range_syntax` feature which I suspect the issue comes from).

**Comment by valeriansaliou** [comment]
Bumped `rs-card-validate` to [06a7c5fa95173cf421888f9df6edc3e5d4519947](https://github.com/valeriansaliou/rs-card-validate/commit/06a7c5fa95173cf421888f9df6edc3e5d4519947).
